09-typescript-in-react/src/exercise-1/blog.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Exercise:
+// * Change extension of this file to .jsx and restart the development server
+// * Define suitable "BlogPost" type that will hold information about id, title and text
```

Comment:
is this a trick question? the props contain 'header' for title and 'body' for text,, not sure if people will get too confused :)

Reply:
Oh, good catch, I changed it and I forgot to update the instructions. I will fix it, thank you 👍

Reply:
I remember now, I actually left it there on purpose (there is a note later on in instructions "WARNING: there is a bug that will emerge with using a proper typing, find it and fix it."), but you are right that it might be too confusing.